samples/GalleryApp/MainPage.xaml

```diff
@@ -9,57 +9,67 @@
     mc:Ignorable="d"
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 
-    <Grid>
+    <Page.Resources>
+        <DataTemplate x:Key="ImageGridView_DefaultItemTemplate"
+                      x:DataType="local:Skill">
+            <Grid Background="LightGray" Height="400" Width="400">
+                <Grid Height="200" Width="400" VerticalAlignment="Bottom" 
+                      HorizontalAlignment="Center">
+                    <Rectangle Height="200" Width="400" VerticalAlignment="Bottom" 
+                               Fill="Gray" Opacity="0.25"/>
+                    <TextBlock Text="{x:Bind Name}" Padding="10, 0, 10, 0"
+                               Style="{StaticResource SubheaderTextBlockStyle}"/>
+                    <TextBlock Text="{x:Bind Description}" Padding="10, 60, 10, 0"
+                               Style="{StaticResource BodyTextBlockStyle}"/>
+                </Grid>
+            </Grid>
```

Comment:
Will investigate on how to get data binding work with user control.
Task 22513680: Investigate how data binding work with user control (Thumbnail.xaml) inside DataTemplate

Reply:
please add this as a comment in the code

---
In reply to: [299718029](https://github.com/microsoft/WindowsVisionSkillsPreview/pull/45#discussion_r299718029) [](ancestors = 299718029)